fabtests/common/shared.c

```diff
@@ -1376,17 +1472,22 @@ static void ft_close_fids(void)
 
 void ft_free_res(void)
 {
-	ft_close_fids();
+	if (tx_mr_bufs)
+		ft_cleanup_mr_array(tx_ctx_arr, tx_mr_bufs);
+	if (rx_mr_bufs)
+		ft_cleanup_mr_array(rx_ctx_arr, rx_mr_bufs);
```

Comment:
nit: You can move the if (ctx_arr) check into the cleanup call and save a whole line of code!

Reply:
Wow! A whole line!